syweb/webclient/app/settings/settings-controller.js

```diff
@@ -38,6 +38,20 @@ angular.module('SettingsController', ['matrixService', 'mFileUpload', 'mFileInpu
         matrixService.saveConfig();
         $scope.config = matrixService.config();
     };
+
+    function fetchRules() {
+        notificationService.getGlobalRulesets().then(function(rulesets) {
+            $scope.settings.rules = rulesets;
+        });
+    };
+
+    $scope.content_rule_add_action = 'notify';
+    $scope.room_rule_add_action = 'notify';
+    $scope.sender_rule_add_action = 'notify';
```

Comment:
Minor: I would prefer this to be namespaced to avoid variable pollution: e.g.
```
$scope.rule_add_action = {
  content: "notify",
  room: "notify",
  sender: "notify"
};
```
Which can then just be dotted in the html like ``ng-model="rule_add_action.room"``